invenio_vocabularies/contrib/affiliations/schema.py

```diff
@@ -59,3 +59,12 @@ class AffiliationRelationSchema(ContribVocabularyRelationSchema):
     ftf_name = "name"
     parent_field_name = "affiliations"
     name = SanitizedUnicode()
+    identifiers = IdentifierSet(
+        fields.Nested(
+            partial(
+                IdentifierSchema,
+                allowed_schemes=affiliation_schemes,
+                identifier_required=False,
+            )
+        )
+    )
```

Comment:
```suggestion
    identifiers = IdentifierSet(
        fields.Nested(
            partial(
                IdentifierSchema,
                allowed_schemes=affiliation_schemes,
                identifier_required=False,
            )
        ),
        dump_only=True,
    )
```

minor: just to make sure we don't accept it as input.